openapi-mock/server/mock.js

```diff
@@ -17,18 +17,18 @@ module.exports = async function (config) {
     var app = express()
     for (var i = 0; i < config.apis.length; i++) {
         var api = config.apis[i];
-        if (api.type = "openapi") {
+        if (!api.type || api.type == "openapi") {

```

Comment:
The changes in this file causes the errors. Openapi errors are failing and probably affects the functionality of stress mock. When I reverted this change, open api tests passed, and most of the stress mock tests passed. I assume the error is happening because there was a typo in the previous version `api.type = "openapi"` with one `=` and caused us to write either wrong tests, or wrong functionality.

Reply:
the thing is that this is something wrong in the code and now it's fixed. I am sure that the tests are right now and still the main problem is that the **tests pass with npm test but don't when we use lerna** ("that is the issue we want to solve")

Reply:
this wasn't the problem